File: easy/1. Two Sum/csharp/solution.cs
class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for (int i = 0; i < nums.Length; i++) {
            for (int j = i + 1; j < nums.Length; j++) {
                if (nums[i] + nums[j] == target) {
                    return new int[]{i, j};
                }
            }
        }
        return new int[]{};
    }
}
class Program {
    static void Main(string[] args) {
        Solution test = new Solution();
        int[] nums = {2, 7, 11, 15};
        int target = 9;
        int[] result = test.TwoSum(nums, target);
        Console.WriteLine("[" + result[0] + ", " + result[1] + "]");
    }
}

File: easy/1. Two Sum/python/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        for i in range(len(nums)):
            for j in range(i+1, len(nums)): #start at i+1 to avoid duplication
                if (nums[i] + nums[j]) == target:
                    return [i,j]

if __name__ == '__main__':
    test = Solution()
    nums = [2,7,11,15]
    target = 9
    print(test.twoSum(nums, target))